rust/worker/src/execution/operators/merge_knn_results.rs

```diff
@@ -160,7 +182,7 @@ fn merge_results(
     hnsw_result_vectors: &Option<Vec<Vec<f32>>>,
     brute_force_result_user_ids: &Vec<String>,
     brute_force_result_distances: &Vec<f32>,
-    brute_force_result_vectors: &Option<Vec<Vec<f32>>>,
+    brute_force_result_vectors: &Vec<Vec<f32>>,
```

Comment:
any reason why this should be an Option instead?

Reply:
Seems fine given that the bf operator never returns option